web/src/components/vis/HeatmapTile.vue

```diff
@@ -118,19 +127,41 @@ vis-tile-large(v-if="plot", title="Heatmap", expanded, download, :download-impl=
     sample-colorer(:dataset="dataset", v-model="sampleColor")
 
     v-toolbar.darken-3(color="primary", dark, flat, dense)
-      v-toolbar-title Dendogram
+      v-toolbar-title.switch-title Dendrogram
     v-card.mx-3(flat)
       v-card-actions(:style="{display: 'block'}")
-        v-checkbox.my-0(v-model="row.dendogram", label="Metabolite", hide-details)
-        v-checkbox.my-0(v-model="column.dendogram", label="Sample", hide-details)
+        v-checkbox.my-0(
+            v-model="row.dendrogram",
+            label="Metabolite",
+            hide-details)
+        v-checkbox.my-0(
+            v-model="column.dendrogram",
+            label="Sample",
+            hide-details)
     toolbar-option(title="Layout", :value="layout",
         :options="layouts",
         @change="layout = $event")
 
   heatmap(ref="heatmap",
       v-if="plot && plot.data && dataset.ready && values",
-      :values="values", transposed,
+      :values="values",
+      :original="original",
+      transposed,
       :column-config="column", :row-config="row", :layout="layout",
       :row-clustering="plot.data ? plot.data.column : null",
       :column-clustering="plot.data ? plot.data.row : null")
 </template>
+
+<style lang="scss" scoped>
+.switch-title {
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  overflow: visible;
+  width: 100%;
+  .switch {
+    flex-grow: 0;
+    margin-right: -10px;
+  }
+}
+</style>
```

Comment:
Since you removed this switch, should this CSS be removed?

Reply:
Good catch--see 0030e51.